docs/try-viam/rover-resources/rover-tutorial-fragments.md

```diff
@@ -53,28 +53,10 @@ Typically, this would suggest that they should be configured as enable pins, but
 
 {{% /alert %}}
```

Comment:
```suggestion
{{% /alert %}}

```suggestion
- An [accelerometer](/components/movement-sensor/adxl345/)

For information about how you would configure a component yourself if you weren't using the fragment, click the links on each component above.
To see the configured pin numbers and other values specific to this fragment, [see the components on the configuration page](#see-the-components-on-the-configuration-page).
```
```